### PR TITLE
[fix] Compatible with MongoDB versions earlier than 3.6

### PR DESCRIPTION
**Pull request by gjjjj0101**
## What's changed?

link [#1692](https://github.com/apache/hertzbeat/issues/1692)

## Checklist

- [x]  I have read the [Contributing Guide](https://hertzbeat.apache.org/docs/community/code_style_and_quality_guide)
- [ ]  I have written the necessary doc or comment.
- [x]  I have added the necessary unit tests and all cases have passed.

## Add or update API

- [ ] I have added the necessary [e2e tests](https://github.com/apache/hertzbeat/tree/master/e2e) and all cases have passed.

### CODE REVIEW

File: collector/src/main/java/org/apache/hertzbeat/collector/collect/mongodb/MongodbSingleCollectImpl.java [hunk 1]
```diff
@@ -112,20 +111,18 @@ public void collect(CollectRep.MetricsData.Builder builder, long monitorId, Stri
...
             MongoDatabase mongoDatabase = mongoClient.getDatabase(metrics.getMongodb().getDatabase());
-            clientSession = mongoClient.startSession();
             CollectRep.ValueRow.Builder valueRowBuilder = CollectRep.ValueRow.newBuilder();
             Document document;
             if (metricsParts.length == 1) {
-                document = mongoDatabase.runCommand(clientSession, new Document(command, 1));
+                document = mongoDatabase.runCommand(new Document(command, 1));
             } else {
-                document = mongoDatabase.runCommand(clientSession, new Document(command, 1));
+                document = mongoDatabase.runCommand(new Document(command, 1));
```
Comment (crossoverJie) [inline collector/src/main/java/org/apache/hertzbeat/collector/collect/mongodb/MongodbSingleCollectImpl.java:125]: Why not combine lines 123 and 125 into one line and put it outside the `if` statement?

Comment (gjjjj0101) [reply collector/src/main/java/org/apache/hertzbeat/collector/collect/mongodb/MongodbSingleCollectImpl.java]: Thanks, I refactor it.

File: collector/src/main/java/org/apache/hertzbeat/collector/collect/mongodb/MongodbSingleCollectImpl.java [hunk 2]
```diff
@@ -112,23 +111,16 @@ public void collect(CollectRep.MetricsData.Builder builder, long monitorId, Stri
...
         CacheIdentifier identifier = null;
         try {
             identifier = getIdentifier(metrics.getMongodb());
             mongoClient = getClient(metrics, identifier);
             MongoDatabase mongoDatabase = mongoClient.getDatabase(metrics.getMongodb().getDatabase());
-            clientSession = mongoClient.startSession();
             CollectRep.ValueRow.Builder valueRowBuilder = CollectRep.ValueRow.newBuilder();
-            Document document;
-            if (metricsParts.length == 1) {
-                document = mongoDatabase.runCommand(clientSession, new Document(command, 1));
```
Comment (tomsun28) [inline collector/src/main/java/org/apache/hertzbeat/collector/collect/mongodb/MongodbSingleCollectImpl.java:126]: hi, if do not use clientSession close, is the mongoDatabase will auto close the seesion after run command?

Comment (gjjjj0101) [reply collector/src/main/java/org/apache/hertzbeat/collector/collect/mongodb/MongodbSingleCollectImpl.java:126]: mongoDatabase is used to encapsulate the functions for users to operate the database, and construct the real execution options and hand it over to mongoClient for execution, so it does not need to close.

**Review by tomsun28 — APPROVED** [review]
👍LGTM

**Pull request merged**